docs/apidoc/watertap.rst

```diff
@@ -8,7 +8,7 @@ Subpackages
    :maxdepth: 4
 
    watertap.edb
-   watertap.flowsheets
+   watertap.examples.flowsheets
```

Comment:
Given that we are expecting more things under examples, would it be better to have a new `examples.rst` file here which for now is just a TOC pointing onwards to `flowsheets`?

Reply:
I'm honestly not familiar with the apidoc. I can't find it on the webpage, but yes the TOC should be for examples. Since I don't know how to view the change right now, I'll leave it for this PR.